lib/data/schema.js

```diff
@@ -78,6 +81,9 @@ const _recurseFormFields = (instance, bindings, repeats, selectMultiples, envelo
       const binding = bindings.find((bind) => absolutify(bind.nodeset, envelope) === bindingPath);
       if (binding != null) {
         field.type = binding.type || 'unknown'; // binding should have a type.
+        const prop = saveToAttr(binding);
+        if (prop)
+          field.entity_prop = binding[prop];
```

Comment:
`entity_prop` -> `propertyName`
will need to put in field aux in that other place in the code where this gets used